### ISSUE TITLE
✅ 🔗 🐞 Bug Report: New Swiper Library (v8.45.) breaks the Elementor Lightbox (Related to #21384) - [ED-9856]

### ISSUE THREAD

**Issue by dara-mtl** [elementor/elementor#21413]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Activating the new library under the features will break the light box layout. Image are not centered anymore and are either at the top or the bottom.

Also, a file is missing from the library: .../wp-content/plugins/elementor/assets/lib/swiper/v8/swiper-bundle.min.js.map:

### Steps to reproduce

1. Go under Features and enable "Upgrade Swiper Library"
2. Open any image in the lightbox
3. For the missing library: simply look at the devtools under the console tab

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.6.10
	PHP Version: 8.1.16
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://asiabettinglist.com
	Home URL: https://asiabettinglist.com
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email:
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.6
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.6.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	CMB2
		Version: 2.10.1
		Author: CMB2 team

	CMB2 Field Type: Attached Posts
		Version: 1.2.7
		Author: WebDevStudios

	Contact Form 7
		Version: 5.7.4
		Author: Takayuki Miyoshi

	Custom Post Type UI
		Version: 1.13.4
		Author: WebDevStudios

	Elementor
		Version: 3.11.1
		Author: Elementor.com

	Elementor CWM Extensions
		Version: 1.0.0
		Author: Cambodia Webmaster

	Elementor CWM Widgets
		Version: 1.0.2
		Author: Cambodia Webmaster

	Elementor Header & Footer Builder
		Version: 1.6.13
		Author: Brainstorm Force, Nikhil Chavan

	Git Updater
		Version: 12.1.2
		Author: Andy Fragen

	White Label CMS
		Version: 2.5
		Author: www.videousermanuals.com


== Must-Use Plugins ==
	1&1 Product Subdomain
		Version: 1.1.0
		Author: 1&1


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive
	Hello Theme Header & Footer: Inactive
	Editor V2: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive
	Save as Default: Active by default


== Log ==
	
PHP: showing 7 of 7PHP: 2023-02-06 08:36:18 [warning X 54][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor/app/modules/import-export/module.php::628] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-06 08:41:31 [warning X 1][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-07 05:38:19 [warning X 1][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor-cwm-widgets/widgets/post-widget.php::3169] foreach() argument must be of type array|object, null given [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-07 05:38:21 [notice X 795][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor-cwm-extension/custom-css.php::103] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-07 06:33:15 [warning X 3][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor-cwm-extension/tags/post-content.php::41] Undefined variable $post_content [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-07 06:49:08 [warning X 1][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor-cwm-extension/tags/post-content.php::40] Attempt to read property "ID" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-14 09:49:33 [warning X 1][/homepages/17/d950710189/htdocs/clickandbuilds/asiabettinglist/wp-content/plugins/elementor-cwm-widgets/widgets/post-widget.php::3256] Array to string conversion [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 5 of 5JS: 2023-02-06 08:41:02 [error X 10][https://asiabettinglist.com/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.10.2:2:2932] elementor_new_template_form_controls is not defined 
JS: 2023-02-06 08:59:32 [error X 26][https://asiabettinglist.com/wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-core,jquery-migrate,underscore,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:2:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-02-13 12:03:07 [error X 1][https://asiabettinglist.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0:3:810235] L.select2 is not a function 
JS: 2023-02-15 02:20:12 [error X 1][https://asiabettinglist.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.0:3:592136] Cannot read properties of null (reading 'id') 
JS: 2023-02-20 10:41:02 [error X 5][https://asiabettinglist.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view91") has already been destroyed and cannot be used. 

Log: showing 9 of 92023-02-13 09:42:32 [info] elementor::elementor_updater Started 
2023-02-13 09:42:32 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-13 09:42:32 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-13 09:42:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-13 09:42:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-15 21:43:04 [info] elementor::elementor_updater Started 
2023-02-15 21:43:04 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-15 21:43:04 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-15 21:43:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]



== Elementor - Compatibility Tag ==
	
	Elementor CWM Extensions: Compatibility not specified
	Elementor CWM Widgets: Compatibility not specified
	Elementor Header & Footer Builder: Incompatible
				
````

</details>

**Comment by nicholaszein** [comment]
Hello @dara-mtl! 

Thank you for submitting your issue! 🙏 

We can recreate this issue and we are already working on a fix. 💪 😉 

Kind regards

**Comment by jeromevalette1** [comment]
This CSS make it better but the click event on elementor-swiper-button doesn't work.

.elementor-lightbox .swiper {
    height: 100%;
}
.elementor-lightbox .elementor-swiper-button-next, .elementor-lightbox .elementor-swiper-button-prev {
    height: 100%;
    display: flex;
    align-items: center;
    width: 15%;
    justify-content: center;
    font-size: var(--lightbox-navigation-icons-size);
}

.elementor-lightbox  .elementor-swiper-button {
    position: absolute;
    display: inline-flex;
    z-index: 1;
    cursor: pointer;
    font-size: 25px;
    color: hsla(0,0%,93.3%,.9);
    top: 50%;
    transform: translateY(-50%);
}

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor **v3.11.3**! 🥳 

✅ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
